docs/api/sm-api.md

```diff
@@ -245,6 +245,11 @@ Similarly, to sort the results in a descending order prefix the field name with
 ?sort=-field-name
 ```
 
+To sort on multiple fields, use the following syntax:
+```
+sort[]=field-name1&sort[]=field-name2
```

Comment:
It's also not clear from this that you can sort by order, maybe it would be more concise to make one of them a date field and pass in `-date_field`